### DISCUSSION TITLE
flutter pag内置kotlin-gradle-plugin版本过低

### DISCUSSION THREAD

**Discussion by llqidian** [Tencent/libpag#1728] [General] [answered]
The Android Gradle plugin supports only Kotlin Gradle plugin version 1.5.20 and higher.
The following dependencies do not satisfy the required version:
project ':pag' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.20

**Comment by domchen** [comment, marked as answer]
Flutter的插件由第三方团队维护，建议到对应的仓库直接提issue。
